content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -2332,6 +2332,111 @@ Starting in v1.16, this annotation was removed in favor of
 [Pod Priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/).
 {{< /note >}}
 
+### jobset.sigs.k8s.io/jobset-name
+
+Type: Label, Annotation
+
+Example:  `jobset.sigs.k8s.io/jobset-name: "my-jobset"`
+
+Used on: Jobs, Pods
+
+This label/annotation is used to store the name of the JobSet that a Job or Pod belongs to.
+[JobSet](https://jobset.sigs.k8s.io) is an extension API that you can deploy into your Kubernetes cluster.
+
+### jobset.sigs.k8s.io/replicatedjob-replicas
+
+Type: Label, Annotation
+
+Example: `jobset.sigs.k8s.io/replicatedjob-replicas: "5"`
+
+Used on: ReplicatedJobs
+
+This label/annotation specifies the number of replicas for a ReplicatedJob.
+
+### jobset.sigs.k8s.io/replicatedjob-name
+
+Type: Label, Annotation
+
+Example: `jobset.sigs.k8s.io/replicatedjob-name: "my-replicatedjob"`
+
+Used on: Jobs, Pods
+
+This label or annotation stores the name of the replicated job that this Job or Pod is part of.
+
+### jobset.sigs.k8s.io/job-index
+
+Type: Label, Annotation
+
+Example: `jobset.sigs.k8s.io/job-index: "0"`
+
+Used on: Jobs, Pods
+
+This label/annotation is set by the JobSet controller on child Jobs and Pods. It contains the index of the Job replica within its parent ReplicatedJob.
+
+### jobset.sigs.k8s.io/job-key
+
+Type: Label, Annotation
+
+Example: `jobset.sigs.k8s.io/job-key=0f1e93893c4cb372080804ddb9153093cb0d20cefdd37f653e739c232d363feb`
```

Comment:
```suggestion
Example: `jobset.sigs.k8s.io/job-key: 0f1e93893c4cb372080804ddb9153093cb0d20cefdd37f653e739c232d363feb`
```

This doesn't look right. Is `0f1e93893c4cb372080804ddb9153093cb0d20cefdd37f653e739c232d363feb` definitely a valid label value? It looks longer than is allowed.

Reply:
It is, we set the value as the SHA256 hash of the namespaced job name [here](https://github.com/kubernetes-sigs/jobset/blob/b92cbdcbe77f168a82cf12c9f349bdd053d2ae5c/pkg/controllers/jobset_controller.go#L733).

Reply:
OK, but a SHA256 hash is 64 UTF-8 (or ASCII) characters, and conformant label values are [documented](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) as being up to 63 characters long.

Something doesn't add up. Maybe there's an off-by-one in the API server (I hope not).

Reply:
Sorry, it's SHA1 not SHA256, my mistake. So it's only 40 characters.

Reply:
OK. Minor docs bug then; we can merge this with or without a fix (still would need a post-merge fixup).